src/mina.ts

```diff
@@ -0,0 +1,366 @@
+            "4656175953888995612264371467596648522808911819700660048695373348629527757049",
+            "23221574237857660318443567292601561932489621919104226163978909845174616477329"
+        ],
+        [
+            "1878392460078272317716114458784636517603142716091316893054365153068227117145",
+            "2370412714505757731457251173604396662292063533194555369091306667486647634097",
+            "17409784861870189930766639925394191888667317762328427589153989811980152373276"
+        ],
+        [
+            "25869136641898166514111941708608048269584233242773814014385564101168774293194",
+            "11361209360311194794795494027949518465383235799633128250259863567683341091323",
+            "14913258820718821235077379851098720071902170702113538811112331615559409988569"
+        ],
+        [
+            "12957012022018304419868287033513141736995211906682903915897515954290678373899",
+            "17128889547450684566010972445328859295804027707361763477802050112063630550300",
+            "23329219085372232771288306767242735245018143857623151155581182779769305489903"
+        ],
+        [
+            "1607741027962933685476527275858938699728586794398382348454736018784568853937",
+            "2611953825405141009309433982109911976923326848135736099261873796908057448476",
+            "7372230383134982628913227482618052530364724821976589156840317933676130378411"
+        ],
+        [
+            "20203606758501212620842735123770014952499754751430660463060696990317556818571",
+            "4678361398979174017885631008335559529633853759463947250620930343087749944307",
+            "27176462634198471376002287271754121925750749676999036165457559387195124025594"
+        ],
+        [
+            "6361981813552614697928697527332318530502852015189048838072565811230204474643",
+            "13815234633287489023151647353581705241145927054858922281829444557905946323248",
+            "10888828634279127981352133512429657747610298502219125571406085952954136470354"
+        ]
+    ],
+    "fullRounds": 55,
+    "partialRounds": 0,
+    "hasInitialRoundConstant": false,
+    "stateSize": 3,
+    "rate": 2,
+    "power": 7
+};
+
+const MinaField = mod.Field(p)
+
+export const NoblePoseidonMina = poseidon({
+    Fp: MinaField,
+    t: poseidonParamsKimchiFp["stateSize"],
+    roundsFull: poseidonParamsKimchiFp["fullRounds"],
+    roundsPartial: poseidonParamsKimchiFp["partialRounds"],
+    sboxPower: poseidonParamsKimchiFp["power"],
+    mds: poseidonParamsKimchiFp["mds"].map((row) => {
+        return row.map((x) => BigInt(x))
+    }),
+    roundConstants: poseidonParamsKimchiFp["roundConstants"].map((row) => {
+        return row.map((x) => BigInt(x))
+    })
+});
+
+const Point = mina_pallas.ProjectivePoint
+const H = Point.fromAffine(H_mina_pallas)
+
+export const NoblePedersenMina = (x: bigint, r: bigint): bigint => {
+    const commitment = Point.BASE.multiply(x).add(H.multiply(r))
```

Comment:
How do we knows that H_mina_pallas is independent of Point.BASE? Where does it come from?